crates/sparrow-physical/src/plan.rs

```diff
@@ -0,0 +1,12 @@
+use crate::Step;
+
+/// A plan is a directed, acyclic graph of steps.
+///
+/// The plan is represented as an array of steps, with each step referencing
+/// it's children (inputs) by index. The array is topologically sorted so that
```

Comment:
topology here being the order of visitation (or evaluation?).

Reply:
Rather than trying to explain "order of visitation" just stated: "every step appears after the inputs to that step". This seems to explain what we mean by "topologically sorted" by stating the exact property, which seems preferable. Otherwise, we'd need to explain what we mean by "order of visitation" (DFS or BFS? bottom up or top down? etc.) Thoughts?